### DISCUSSION TITLE
Error Generating report

### DISCUSSION THREAD

**Discussion by oarguido** [Data-Centric-AI-Community/fg-data-profiling#1444] [Q&A]
Hi guys,

I have been getting this error using ydata-profiling:

TypeCheckError: argument "config_file" (None) did not match any element in the union:
  pathlib.Path: is not an instance of pathlib.Path
  str: is not an instance of str

I'm still new to python and programming. I've already reinstalled the library but the problem persists... no idea what the error message means.

Can anyone help?

Thank you!

EDIT: this was working perfectly a few weeks ago.

**Comment by hug1thub** [comment]
Hi, i have just installed the package and i have the same problem

**Reply by oarguido** [reply]
I had to downgrade my matplotlib to the previous version (3.7.3) and it started working again. Hope it helps!
